src/VCEL.CSharp/Expression/Func/DefaultCSharpFunctions.cs

```diff
@@ -63,21 +63,27 @@ public DefaultCSharpFunctions()
             switch (args.Length)
             {
                 case 2:
-                    {
-                        var sourStr = args[0]?.ToString();
-                        var startIndex = int.Parse(args[1]?.ToString());
-                        return $"{sourStr}.Substring({startIndex})";
-                    }
+                {
+                    var sourStr = args[0]?.ToString();
```

Comment:
I assume it meant to be source?
```suggestion
                    var sourceString = args[0]?.ToString();
```